simulation/sub8_gazebo/nodes/gazebo_controller.py

```diff
@@ -66,6 +71,16 @@ def publish_height(self, msg):
             )
         )
 
+    def publish_blueview_ping(self, msg):
+        self.blueview_ranges_pub.publish(
+            BlueViewPing(
+                header=msg.header,
+                bearings=[(msg.angle_min + i * msg.angle_increment) for i in xrange(len(msg.ranges))],
+                ranges=msg.ranges,
+                intensities=[500 for x in msg.ranges]
```

Comment:
When I was setting the blueview intensities to be the laser intestines, the pointcloud wouldn't show up in rviz because all the intensities would be 0. Setting it to ranges is better than just a constant value I think, it it makes it a rainbow! The intensities aren't being used anywhere in the ogrid stack.